tests/lib/location/prj.conf

```diff
@@ -16,6 +16,9 @@ CONFIG_LOCATION_METHOD_CELLULAR=y
 CONFIG_LOCATION_SERVICE_HERE=y
 CONFIG_LOCATION_SERVICE_HERE_API_KEY="MyApiKey"
 
+# Increase AT monitor heap because %NCELLMEAS notifications can be large
+CONFIG_AT_MONITOR_HEAP_SIZE=512
```

Comment:
I think that the requirements for this should be considered to be documented. Seems that also location lib sample is having same config for this and based on my knowledge from the past, 512 bytes ain't large after all, especially when now using NCELLMEAS=3,15 in cellular location algorithm. 
I recall that the theoretical max for NCELL meas response is ~4k bytes and that's why for I wrote that comment on mosh config:  https://github.com/nrfconnect/sdk-nrf/blob/main/samples/cellular/modem_shell/prj.conf#L87-L90

Reply:
Internal information shows that 4020 is theoretical maximum. In practice the maximum is 1654 because GCI cells never have neighbor cell information. And actually it's 1485 bytes because serving cell will neither have normal neighbors with GCI search.
I'll make a separate PR about the CONFIG_AT_MONITOR_HEAP_SIZE related configs in different apps/samples and asserting proper size in Location library.